scripts/generate-google3-docs/resource-reference/templates/secretmanager_secretmanagersecret.tmpl

```diff
@@ -49,6 +49,10 @@
 </tbody>
 </table>
 
+## Prerequisites
+Before you can use this resource, assign the Secret Manager Admin role (roles/secretmanager.admin) on the project, folder, or organization.
```

Comment:
Could you add a new line in between the title and the paragrah?

Reply:
Could you also run the mdlint tool for the .tmpl and .md files (hopefully they work?) to ensure the generated .md files pass the lint checks internally?

Reply:
Done